geniuspaste/src/geniuspaste.c

```diff
@@ -102,6 +102,10 @@ PLUGIN_SET_TRANSLATABLE_INFO(LOCALEDIR, GETTEXT_PACKAGE, PLUGIN_NAME,
                              PLUGIN_VERSION, "Enrico Trotta <enrico.trt@gmail.com>")
 
 
+static void show_msgbox(GtkMessageType type, GtkButtonsType buttons,
```

Comment:
This is to trick cppcheck: it complains that it doesn't know `G_GNUC_PRINTF` when it is used in the function definition. But it is ok (probably ignored) if it is used in the function declaration.

This macro is used in other places in G-P in function declarations as well, so I guess it is fine.